server/paymentProviders/stripe/virtual-cards.js

```diff
@@ -121,6 +120,26 @@ const setCardStatus = async (virtualCard, status = 'canceled' | 'active' | 'inac
   return data;
 };
 
+const sendPurchaseNotifyEmail = async (virtualCard, amount, currency) => {
+  const collectiveId = virtualCard.collective.id;
+  const collective = await models.Collective.findByPk(collectiveId);
+  const user = await models.User.findByPk(virtualCard.UserId);
+  const responsibleAdmin = await models.Collective.findByPk(user.CollectiveId);
+
+  const adminUsers = await collective.getAdminUsers();
+
+  await emailLib.send(
+    activities.VIRTUAL_CARD_PURCHASE,
+    adminUsers.map(u => u.dataValues.email),
+    {
+      responsibleAdmin: responsibleAdmin.dataValues,
+      collective: collective.dataValues,
+      amount,
+      currency,
+    },
+  );
+};
```

Comment:
I think this looks good. But for consistency, I would add a `VIRTUAL_CARD_PURCHASE` activity handler in `lib/notifications/email.js`.

Reply:
Done. 👍🏽